addOns/webuipoc/src/main/pocs/reactWebUI/src/App.js

```diff
@@ -2,9 +2,11 @@
 import './App.css';
```

Comment:
These changes should be removed form this PR.

Reply:
To be clear, I mean the whole UI changes that are already being done in the other PR.